lib/stat_tracker.rb

```diff
@@ -11,7 +11,7 @@ def self.from_csv(location_path)
    end
 
    def initialize(locations_path)
-      @data_game = read_games_csv(locations_path[:games])
+      @data_games = read_games_csv(locations_path[:games])
```

Comment:
This is great! it is an array so it's best practice to name variable in plural! great job!